File: testing/driver/src/main/java/net/corda/testing/driver/impl/DriverInternalDSL.java
package net.corda.testing.driver.impl;

import net.corda.testing.driver.DriverDSL;
import net.corda.testing.driver.Framework;
import org.jetbrains.annotations.NotNull;

/**
 * This internal interface is package private,
 * to keep it hidden from Java code.
 */
interface DriverInternalDSL extends DriverDSL {
    @NotNull
    Framework getFramework();
}

File: testing/driver/src/main/kotlin/net/corda/testing/driver/impl/DriverInternalDSL.kt
package net.corda.testing.driver.impl

import net.corda.testing.driver.DriverDSL
import net.corda.testing.driver.Framework
import net.corda.v5.base.types.MemberX500Name

interface DriverInternalDSL : DriverDSL {
    fun getFramework(x500Name: MemberX500Name): Framework
}
